### PR TITLE
Update Druid Console docs for 0.15.0

### PR DESCRIPTION
**Pull request by vogievetsky**
Updating the docs for the Druid console with the console as it will look in 0.15.0.
This PR is already out of date for master but should be merged and backported to `0.15.0`

Note that the annotations come from here: https://docs.google.com/presentation/d/1D0FRf46OP7bwU2NexkuNuIEui-m7GxPHs4y0j3sBNoc/edit#slide=id.g54ca31720e_0_15

### CODE REVIEW

File: docs/content/operations/druid-console.md [hunk 1]
```diff
@@ -45,49 +45,69 @@ The home view provides a high level overview of the cluster. Each card is clicka
...
 ![home-view](./img/01-home-view.png)
 
+## Data loader
+
+The data loader view allows you to load data by building an ingestion spec with a step-by-step wizard. 
+
+![data-loader-1](./img/02-data-loader-1.png)
+
+After picking the source of your data just follow the series for steps that will show you incremental previews of the data as it will be ingested.
+After filling in the requried details on every step you can navigate to the next step by clicking the `Next` button.
```
Comment (fjy) [inline docs/content/operations/druid-console.md]: required*

File: docs/content/operations/druid-console.md [hunk 2]
```diff
@@ -45,49 +45,69 @@ The home view provides a high level overview of the cluster. Each card is clicka
...
+You can also freely navigate between the steps from the top navigation.
```
Comment (fjy) [inline docs/content/operations/druid-console.md]: I'd add a comment that some steps will only populate if you click on the "Next button" from the previous step

File: docs/content/operations/druid-console.md [hunk 3]
```diff
@@ -45,49 +45,69 @@ The home view provides a high level overview of the cluster. Each card is clicka
...
 ## Servers
 
 The data servers tab lets you see the current status of the historical nodes and MiddleManager (indexer) processes. Note that currently only historical nodes that are actively serving segments will be shown in this view.
 
-![servers](./img/08-servers.png)
+![servers](./img/10-servers.png)
+
+## Query
+
+The query view lets you issue direct DruidSQL queries and display the results as a simple table.
```
Comment (fjy) [inline docs/content/operations/druid-console.md]: Link to SQL docs

File: docs/content/operations/druid-console.md [hunk 4]
```diff
@@ -45,49 +45,69 @@ The home view provides a high level overview of the cluster. Each card is clicka
...
+
+![query-sql](./img/11-query-sql.png)
 
-## SQL
+The query view can also issue queries in Druid's native query format, which is JSON over HTTP.
```
Comment (fjy) [inline docs/content/operations/druid-console.md]: link to native Druid query docs

File: docs/content/operations/druid-console.md (hunk 2 above)
Comment (vogievetsky) [reply docs/content/operations/druid-console.md]: Done

**Pull request merged**